src/dispatch/static/dispatch/src/participant/ParticipantSearchPopover.vue

```diff
@@ -0,0 +1,275 @@
+            >
+              <template v-slot:prepend>
+                <v-avatar
+                  size="14px"
+                  :style="{ background: getAvatarGradient(selectedParticipant) }"
+                ></v-avatar>
+              </template>
+              <span style="font-size: 0.8125rem; font-weight: 500; color: rgb(60, 65, 73)">
+                {{ selectedParticipant }}
+              </span>
+            </v-btn>
+          </template>
+        </FancyTooltip>
+      </template>
+
+      <v-card min-width="200" class="rounded-lg dispatch-side-card">
+        <v-row no-gutters>
+          <v-col align-self="start">
+            <v-text-field
+              v-model="searchQuery"
+              density="compact"
+              variant="solo"
+              single-line
+              hide-details
+              flat
+            >
+              <template v-slot:label>
+                <span class="text-subtitle-2 font-weight-regular"> {{ props.label }} </span>
+              </template>
+            </v-text-field>
+          </v-col>
+          <v-col align-self="end" cols="2" class="pb-2">
+            <Hotkey :hotkey="props.hotkey.toUpperCase()" />
+          </v-col>
+        </v-row>
+        <v-divider></v-divider>
+        <v-list lines="one">
+          <v-list-item
+            v-for="(participant, index) in filteredParticipants"
+            :key="index"
+            @click="selectParticipant(participant)"
+            @mouseover="hoveredParticipant = participant"
+            @mouseleave="hoveredParticipant = ''"
+            density="compact"
+            rounded="lg"
+            class="ml-1 mr-1"
+            active-class="ma-4"
+          >
+            <template v-slot:prepend>
+              <v-avatar
+                class="mr-n2"
+                size="12px"
+                :style="{ background: getAvatarGradient(participant) }"
+              ></v-avatar>
+              <!-- <v-icon class="mr-n6 ml-n2" size="x-small" icon="mdi-account"></v-icon> -->
+            </template>
+            <v-list-item-title class="item-title-font">{{ participant }}</v-list-item-title>
+            <template v-slot:append>
+              <v-icon
+                v-if="participant === selectedParticipant"
+                class="ml-2"
+                size="x-small"
+                :color="hoveredParticipant === selectedParticipant ? 'black' : ''"
+              >
+                mdi-check
+              </v-icon>
+            </template>
+          </v-list-item>
+        </v-list>
+      </v-card>
+    </v-menu>
+  </div>
+</template>
+
+<style scoped>
+.dispatch-side-card {
+  backdrop-filter: blur(12px) saturate(190%) contrast(50%) brightness(130%) !important;
+  border: 0.5px solid rgb(216, 216, 216) !important;
+  border-radius: 8px !important;
+  box-shadow: rgba(0, 0, 0, 0.09) 0px 3px 12px !important;
+  color: rgb(60, 65, 73) !important;
+  opacity: 2 !important;
+}
+
+.item-title-font {
```

Comment:
This has also been defined a bunch of times, we should but them in `styles/index.scss` if they are used across components.